plugins/build.gradle.kts

```diff
@@ -5,11 +5,12 @@ subprojects {
 
     dependencies {
         compileOnly(project(":core"))
-        compileOnly(kotlin("stdlib-jdk8"))
 //        compileOnly(project(":coreDependencies", configuration = "shadow")) // uncomment if IntelliJ does not recognize pacakges from IntelliJ
+        implementation(kotlin("stdlib-jdk8"))
 
+        testImplementation(project(":core"))
+        testImplementation(project(":coreDependencies", configuration = "shadow"))
```

Comment:
I think that's unnecessary as `testApi` has a transitive dependency on those subprojects